web/modules/custom/usagov_top_navigation_menu/js/addMenuStyles.js

```diff
@@ -0,0 +1,34 @@
+(function ($, Drupal, drupalSettings) {
+  /**
+   * @namespace
+   */
+  Drupal.behaviors.mymoduleAccessData = {
+    attach: function (context) {
+      var top_nav = drupalSettings.top_nav_menu;
+      // console.log(top_nav);
+      let dicts = {};
+      for (menu_item in top_nav) {
+        var menu_path = Object.keys(top_nav[menu_item])[0].toString();
+        var menu_id = Object.values(top_nav[menu_item])[0];
+        dicts[menu_path] = menu_id;
+      }
+
+      let current_path = window.location.pathname;
+
+      if (current_path in dicts) {
+        let listItem;
+        listItem = document.getElementById(dicts[current_path]);
+        //check to make sure the description was not left null
+        if (listItem) {
+          let aElem = listItem.getElementsByTagName("a")[0];
+          console.log(`the list item is: ${aElem}`);
```

Comment:
Let's remove (or comment out) the console.log statements before we merge this in.

Reply:
changed!